### PR TITLE
Improving documentation. Adding quick start for different platforms.

### PR DESCRIPTION
**Pull request by gonaumov**
These docker images are wonderful and save a lot of work. I just want to add quick start examples that would be useful to people.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,7 +23,14 @@ docker run --rm \
...
   jekyll build
 ```
-
+#### Quick start under Windows (cmd)
+```
+set new_site_name="my-awesome-site" && docker run --rm --volume="%CD%:/srv/jekyll" -it jekyll/jekyll jekyll new %new_site_name% && cd %new_site_name%
+```
+#### Quick start under Linux / Git Bash
+```sh
+export new_site_name="my-awesome-site" && docker run --rm --volume="$PWD:/srv/jekyll" -it jekyll/jekyll jekyll new $new_site_name && cd $new_site_name
```
Comment (envygeeks) [inline README.md:32]:
```suggestion
export new_site_name="my-awesome-site"
docker run --rm --volume="$PWD:/srv/jekyll" \
  -it jekyll/jekyll jekyll new "$new_site_name"
cd "$new_site_name"
```

Comment (envygeeks) [reply README.md:32]: Is it possible for us to quote variables, and split this out like a proper script they can copy and paste?

Comment (gonaumov) [reply README.md]: Sure. I've changed it. This part `chown -R jekyll /usr/gem/` I've added becase there were user permissions related problems and jekyll was not able to run at all. Please review it again.

**Review by envygeeks — APPROVED** [review]
LGTM!

**Pull request merged**